pkg/kubectl/kubectl_printers/customcolumn.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package printers
+package kubectl_printers
```

Comment:
That package name is wrong! If you're changing it it should be `pkg/kubectl/printers` rather than this.